src/index.ts

```
@@ -36,8 +36,8 @@ export function getAge(
 export function getBirthDate(nrn: NrnInput): Date {
   const { birthDate, serial, checksum } = parse(nrn);
   const partialYear = birthDate.slice(0, 2); // Eg. '86' from '860814'
-  const month = Number(birthDate.slice(2, 4)); // Eg. 8 from '860814'
-  const day = Number(birthDate.slice(4, 6)); // Eg. 14 from '860814'
+  const month = normalizeMonth(birthDate.slice(2, 4)); // Eg. 8 from '860814'
+  const day = Number(birthDate.slice(4, 6)) || 1; // Eg. 14 from '860814'
```

Why `|| 1`?

As I understood it some people can have `00` as birth month. I think it's better to throw an error in that case. Returning a `Date` for them would be ambiguous.

---

Some people can have 00 for the birth month and also 00 for the birth date, so we decided to use the middle of the year and  the first of the month.

---

Yeah so I don't want that. You can't tell the difference between known and unknown dates. Just throw an error and resolve it in your own application. It doesn't make sense for the library to make these kinds of defaults.